swagger.yaml

```diff
@@ -0,0 +1,154 @@
+swagger: "2.0"
+info:
+  description: "Authors Haven - A social platform for the creative at heart."
+  version: "1.0.0"
+  title: "Authors Haven"
+  license:
+    name: "MIT"
+    url: "https://www.apache.org/licenses/LICENSE-2.0.html"
+host: "kingsmen-ah-backend-staging.herokuapp.com"
+basePath: "/api"
+tags:
+- name: "user"
+  description: "User operations"
+schemes:
+  - "https"
+  - "http"
+paths:
+  /users:
+    post:
+      tags:
+      - "user"
+      summary: "Create user"
+      description: "Creates a new user in the application"
+      operationId: "createUser"
+      consumes:
+      - "multipart/form-data"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "username"
```

Comment:
sorry for this, the `in` should be `"body"` not `"query"`

Reply:
Alright @fidelisojeah. Thank you for the feedback. I have implemented it.